src/dev/build/tasks/os_packages/docker_generator/resources/bin/kibana-docker

```diff
@@ -136,7 +136,7 @@ kibana_vars=(
     xpack.reporting.queue.timeout
     xpack.reporting.roles.allow
     xpack.searchprofiler.enabled
-    xpack.security.authProviders
+    xpack.security.authc.providers
```

Comment:
this is a whitelist of sorts, so we'll want to include both the deprecated and undeprecated strings

Reply:
The idea was to drop it completely in master (8.0.0) and include both only in 7.x (7.3.0+) backport. Since it doesn't require any significant effort to migrate I thought it's okay to drop it completely in the next major.

Reply:
ah gotcha sounds good, i just saw the deprecations notice in the app config and made all the assumptions